config/webpack.common.js

```diff
@@ -102,6 +102,7 @@ module.exports = {
       "window.Tether": "tether"
     })
   ],
+  postcss: () => [autoprefixer({browsers: ['last 2 versions']})],
```

Comment:
Eh, after reading through the docs again, there really isn't one. I assumed the default was 'latest version', but its not, its '> 1%, last 2 versions, Firefox ESR' which is more than enough for us ATM. I'll switch to default.